sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample10_SingleCategoryClassify.md

```diff
@@ -0,0 +1,102 @@
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateTextAnalyticsClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+```
+
+## Performing Single Category Classify on one or multiple documents
+
+To perform Custom Category Classification in one or multiple documents, set up an `SingleCategoryClassifyAction` and call `StartAnalyzeActionsAsync` on the documents. The result is a Long Running operation of type `AnalyzeActionsOperation` which polls for the results from the API.
+
+```C# Snippet:TextAnalyticsSingleCategoryClassifyAsync
+// Get input document.
+string document = @"I need a reservation for an indoor restaurant in China. Please don't stop the music. Play music and add it to my playlist.";
+
+// Prepare analyze operation input. You can add multiple documents to this list and perform the same
+// operation to all of them.
+var batchInput = new List<string>
+{
+    document
+};
+
+var singleCategoryClassifyAction = new SingleCategoryClassifyAction(projectName, deploymentName);
+
+TextAnalyticsActions actions = new TextAnalyticsActions()
+{
+    SingleCategoryClassifyActions = new List<SingleCategoryClassifyAction>() { singleCategoryClassifyAction }
+};
+
+// Start analysis process.
+AnalyzeActionsOperation operation = await client.StartAnalyzeActionsAsync(batchInput, actions);
+
+await operation.WaitForCompletionAsync();
+```
+
+The returned `AnalyzeActionsOperation` contains general information about the status of the operation. It can be requested while the operation is running or when it has completed. For example:
+
+```C# Snippet:TextAnalyticsSingleCategoryClassifyOperationStatus
+// View operation status.
+Console.WriteLine($"AnalyzeActions operation has completed");
+Console.WriteLine();
+
+Console.WriteLine($"Created On   : {operation.CreatedOn}");
+Console.WriteLine($"Expires On   : {operation.ExpiresOn}");
+Console.WriteLine($"Id           : {operation.Id}");
+Console.WriteLine($"Status       : {operation.Status}");
+Console.WriteLine($"Last Modified: {operation.LastModified}");
+Console.WriteLine();
+```
+
+To view the final results of the long-running operation:
+
+```C# Snippet:TextAnalyticsSingleCategoryClassifyAsyncViewResults
+// View operation results.
+await foreach (AnalyzeActionsResult documentsInPage in operation.Value)
+{
+    IReadOnlyCollection<SingleCategoryClassifyActionResult> classificationResultsCollection = documentsInPage.SingleCategoryClassifyResults;
+
+    foreach (SingleCategoryClassifyActionResult classificationActionResults in classificationResultsCollection)
+    {
+        if (classificationActionResults.HasError)
```

Comment:
Althought this is great for good practices, it makes the sample so big and it hides the real important part which is the last line lol. line 87.
I will suggest removing it. You can always create an issue to follow up on that idea later if you prefer

Reply:
Sure thing. I agree with the fact that it gets in the way a bit, but I thought it might be better to keep it to remain consistent with the ExtractSummary sample. I'll remove it for now anyways.